plugins/in_tail/tail_config.h

```diff
@@ -85,7 +85,7 @@ struct flb_tail_config {
     int read_from_head;        /* read new files from head     */
     int rotate_wait;           /* sec to wait on rotated files */
     int watcher_interval;      /* watcher interval             */
-    int ignore_older;          /* ignore fields older than X seconds        */
+    int ignore_older;       /* ignore fields older than X seconds        */
```

Comment:
Revert this as well please so the file is not unnecessarily modified by this PR.